kitsune/users/jinja2/users/answers.html

```diff
@@ -0,0 +1,14 @@
+{% extends "users/base_contributions.html" %}
+{% set title = _("{name} | Recent Answers")|f(name=display_name(profile.user)) %}
+{% set canonical_url = canonicalize(viewname="users.answers", username=profile.user.username) %}
+
+{% block contributions %}
+  <h2 class="sumo-callout-heading">{{ _("Recent Answers") }}</h2>
+  <ul class="mzp-u-list-styled">
+    {% for answer in answers %}
```

Comment:
let's add something here when there aren't any answers (similar to what you have done with the questions)